oemer/bbox.py

```diff
@@ -47,14 +52,17 @@ def merge_nearby_bbox(bboxes, distance, x_factor=1, y_factor=1):
     return new_box
 
 
-def rm_merge_overlap_bbox(bboxes, mode='remove', overlap_ratio=0.5):
+def rm_merge_overlap_bbox(
+        bboxes: List[BBox], 
+        mode: str = 'remove', 
+        overlap_ratio: float = 0.5) -> Union[List[BBox], List[BBox], List[BBox]]:
```

Comment:
I'd prefer the following formatting

```
def rm_merge_overlap_bbox(
    bboxes: List[BBox], 
    mode: str = 'remove', 
    overlap_ratio: float = 0.5,
) -> Union[List[BBox], List[BBox], List[BBox]]:
    ...
```
Also notice the additional comma at the end of the last parameter `overlap_ratio`.

Reply:
Sure, it might make sense to configure some auto formatter, as this might speed up the time needed for a PR a lot for both sides. Introducing one now would however explode this PR I think, so for the moment I will do the changes by hand.